test/lit/passes/code-folding_enable-threads.wast

```diff
@@ -83,11 +83,18 @@
   )
  )
  ;; CHECK:      (func $negative-zero-b (result f32)
- ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:  (if (result f32)
  ;; CHECK-NEXT:   (i32.const 0)
- ;; CHECK-NEXT:  )
- ;; CHECK-NEXT:  (block $label$0 (result f32)
- ;; CHECK-NEXT:   (f32.const -0)
+ ;; CHECK-NEXT:   (then
+ ;; CHECK-NEXT:    (block $label$0 (result f32)
+ ;; CHECK-NEXT:     (f32.const -0)
+ ;; CHECK-NEXT:    )
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:   (else
+ ;; CHECK-NEXT:    (block $label$1 (result f32)
+ ;; CHECK-NEXT:     (f32.const -0)
+ ;; CHECK-NEXT:    )
+ ;; CHECK-NEXT:   )
```

Comment:
This was valid before, so this looks like a regression to me? I may be missing the reason you say this change is needed, sorry.